src/NodaTime.Test/TimeZones/IO/IoStream.cs

```diff
@@ -161,9 +161,9 @@ public ReadStreamImpl(IoStream stream)
             /// <returns>
             ///   A long value representing the length of the stream in bytes.
             /// </returns>
-            /// <exception cref="T:System.NotSupportedException">A class derived from Stream does not support seeking. 
```

Comment:
I think these may have been needed at some point by Sandcastle (SHFB?), but if the output documentation still works, then lgtm.